### PR TITLE
Modify the challenge participant page

### PR DESCRIPTION
**Pull request by Sanji515**
#### Changes proposed in this pull request:

- For terms and conditions modal, created a new component as a child of challengeparticipate component

- Link to live demo: http://pr-153-evalai.surge.sh

**<p align="center">Screenshot 1:</p>**

Screenshot from 2019-06-20 23-32-43

**<p align="center">Screenshot 2:</p>**

Screenshot from 2019-06-20 23-31-26

**<p align="center">Screenshot 3:</p>**

Screenshot from 2019-06-20 23-32-01

**<p align="center">Screenshot 4:</p>**

Screenshot from 2019-06-20 23-32-11

**<p align="center">Screenshot 5:</p>**

Screenshot from 2019-06-20 23-32-16

### CODE REVIEW

**Comment by Sanji515** [comment]
@RishabhJain2018 @lunayach @galipremsagar @Shekharrajak 

Please review

File: src/app/app.component.html [hunk 1]
```diff
@@ -5,3 +5,7 @@
 <app-confirm *ngIf="confirmParams['isConfirming']" [params]="confirmParams"></app-confirm>
 <app-modal *ngIf="modalParams['isModalVisible']" [params]="modalParams"></app-modal>
 <app-editphasemodal *ngIf="editPhaseModalParams['isEditPhaseModalVisible']" [params]="editPhaseModalParams"></app-editphasemodal>
+<<<<<<< HEAD
```
Comment (galipremsagar) [inline src/app/app.component.html]: Clean this up. You merged the conflict.

File: src/app/components/challenge/challengeparticipate/challengeparticipate.component.html [hunk 2]
```diff
@@ -1,10 +1,17 @@
-<div class="challenge-participate-container">
-  <div class="inactive" *ngIf="!isActive">
-    Sorry, The Challenge is not active!
+<div class="challenge-card">
+  <div *ngIf="!isActive" class="ev-card-panel ev-md-container card-bt-margin">
```
Comment (galipremsagar) [inline src/app/components/challenge/challengeparticipate/challengeparticipate.component.html:2]: Behaviorally when I'm logged out and try to load a page we are first seeing `Sorry, The Challenge is not active!` message and then switching to `Please Login to access this section.` Ideally we shouldn't be seeing the former message if we are not logged-in in first place.

**Review by galipremsagar — COMMENTED** [review]
Made some first pass of comments.

File: src/app/app.component.html (hunk 1 above)
Comment (Sanji515) [reply src/app/app.component.html]: @galipremsagar done

File: src/app/components/challenge/challengeparticipate/challengeparticipate.component.html (hunk 2 above)
Comment (galipremsagar) [reply src/app/components/challenge/challengeparticipate/challengeparticipate.component.html:2]: Thanks for creating an issue @Sanji515 :https://github.com/Cloud-CV/EvalAI-ngx/issues/181

**Review by galipremsagar — APPROVED** [review]
LGTM other than issue which will be handled in a separate PR.

**Pull request merged**